product-features/token-swaps/key-concepts.md

```diff
@@ -0,0 +1,46 @@
+---
+description: >-
+  Learn the key terms involved in ALEX swap operations.
+---
+
+# Key concepts
+
+### Token Swap
+
+A Token Swap is the exchange or trade of a certain amount of one crypto asset for another. In ALEX Lab Platform, swaps are performed on the ALEX's decentralized exchange (DEX) and are facilitated by liquidity pools.
+
+### Swap Transaction
+
+The ALEX DEX operates through smart contracts built on the Stacks blockchain, so all interactions, including swaps, are carried out through blockchain transactions. When you confirm a swap transaction on the ALEX Lab Platform, you are submitting it to the Stacks network to interact with the ALEX DEX smart contract. Once the transaction is confirmed, it means the swap was executed successfully. If the transaction is reverted, no funds will be lost.
+
+### Base Token
+
+The token you currently hold and want to exchange. Is the token you will transfer to ALEX smart contract during the swap transaction.
```

Comment:
The token you currently hold and want to exchange. This is the token you will transfer to the ALEX smart contract during the swap transaction.

Reply:
Done.